include/triton/Dialect/TritonGPU/IR/Dialect.h

```diff
@@ -72,6 +75,8 @@ SmallVector<unsigned> getOrder(Attribute layout);
 
 bool isaDistributedLayout(Attribute layout);
 
+bool expensiveCat(triton::CatOp cat, Attribute &targetEncoding);
```

Comment:
This is supposed to be in `Transforms/Utility.h` but this function is used at multiple places. Have to clean up dependencies among these helper functions a bit...